packages/backend/package.json

```diff
@@ -88,7 +86,6 @@
         "dev": "nodemon src/index.ts",
         "debug": "nodemon --exec \"node --inspect-brk=0.0.0.0:9229 --require ts-node/register src/index.ts\"",
         "build": "tsc --build tsconfig.json && yarn copy-files",
```

Comment:
We need to remove `copy-files` command from `build`

Reply:
thanks!